services/app-api/serverless.yml

```diff
@@ -61,6 +61,11 @@ provider:
   apiGateway:
     restApiId: ${self:custom.appApiGatewayId}
     restApiRootResourceId: ${self:custom.appApiGatewayRootResourceId}
+  layers:
+    - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-0-1:1
+  tracing:
```

Comment:
is this Amazon X Ray? I saw that referenced a bunch but don't understand what it is.

Reply:
So yes and no -- it's Amazon X-Ray, which is their own tracing platform, but we're not using their X-Ray backend to send the data to. In order to get the auto-instrumentation we turn on active tracing on the lambda, but we configure it to be sent to NR via the OTEL collector. We shouldn't be getting any tracing into the X-Ray managed service itself.